src/analytics/index.tsx

```diff
@@ -52,3 +52,8 @@ export const sendAnalyticsEvent: typeof sendAnalyticsTraceEvent = (event, proper
     sendAnalyticsTraceEvent(event, properties)
   }
 }
+
+// This is only used for initial page load so we can get the user's country
```

Comment:
if you don't want this function to be used in the future (it probably shouldn't), we might want to rename to something like "sendInitializationEvent"